src/style.css

```diff
@@ -0,0 +1,123 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir1", sans-serif;
+  font-size: 16px;
+  font-weight: 900;
+  color: #3d4e61;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+.row {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search {
+  position: relative;
+}
+
+.big_search_wrapper img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.big_search {
+  box-sizing: border-box;
+  font-size: 16px;
+
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  outline: none;
+}
+
+.big_search::placeholder {
+  color: #3d4e61;
+  font-family: inherit;
+  font-weight: 300;
+  text-shadow: none;
+}
+
+.small_search::placeholder {
+  color: #3d4e61;
+  font-family: inherit;
+  font-weight: 300;
+  text-shadow: none;
+}
```

Comment:
you're duplicating a lot of code
read [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 2 - there are bad and good examples of how to write such code in the right way